homeassistant/components/google_photos/strings.json

```diff
@@ -54,6 +54,12 @@
     },
     "api_error": {
       "message": "Google Photos API responded with error: {message}"
+    },
+    "albums_failed": {
+      "message": "Cannot fetch albums from the Google Photos API"
+    },
+    "albums_not_loaded": {
+      "message": "Albums have not yet been loaded from Google Photos API"
```

Comment:
Hmm, Not sure if I like this message because it indicates something went wrong, and that the problem lies on the HA side, but it doesn't include a way to fix it

Reply:
But since we are postponing the coordinator fetch I also don't have any other way to do both the postponing + never make this issue happen

Reply:
An alternative could be to block startup. But it could take say 15 to 30 seconds for a large account. Maybe that's not so bad.

Another option is it could block startup once, and persist the results, then refresh faster next time. It wasn't sure it was worth doing this. Didn't feel worth it to me to do this.

I will continue to see it there are other work arounds to make it faster to see if I can remove this.

Have a recommendation?

Reply:
How about this, to make it simple for now i'll revert it and this will block startup and I can revisit if this seems like a common problem in practice.

Reply:
I think that sounds reasonable, it will at least avoid overengineering something that might not be a big problem in practice